app/views/puppetclasses/_classes_parameters.html.erb

```diff
@@ -2,6 +2,7 @@
   <thead class="white-header">
     <tr>
       <th class='col-md-3'><%= _('Puppet Class') %></th>
+      <th class='col-md-2'><%= _('Type') %></th>
```

Comment:
This width causes the total column to go over 12 which is the default grid size used in bootstrap. Perhaps the width of other columns should be adjusted to fit this in.